lms/static/js/student_account/views/RegisterView.js

```diff
@@ -102,6 +157,90 @@
                     return this;
                 },
 
+                postRender: function() {
+                    var inputs = [
+                            this.$('#register-name'),
+                            this.$('#register-email'),
+                            this.$('#register-username'),
+                            this.$('#register-password'),
+                            this.$('#register-country')
+                        ],
+                        inputTipSelectors = ['tip error', 'tip tip-input'],
+                        onInputFocus = function() {
+                            // Apply on focus styles to input
+                            $(this).prev('label').addClass('focus-in');
+                            $(this).prev('label').removeClass('focus-out');
```

Comment:
You can chain jQuery methods so this can be updated to;
```
 $(this).prev('label').addClass('focus-in')
                      .removeClass('focus-out');
```